box-share-sdk/src/main/java/com/box/androidsdk/share/vm/BaseShareVM.java

```diff
@@ -66,4 +67,21 @@ public void fetchItemInfoFromRemote(BoxItem item) {
     public LiveData<PresenterData<BoxItem>> getItemInfo() {
         return mItemInfo;
     }
+
+
+    /**
+     * Returns user id of the current user.
+     * @return user id of the current user
+     */
+    public String getUserId() {
+        return mShareRepo.getUserId();
+    }
+
+    /**
+     * Returns the avatar controller for displaying collaborators' avatars.
+     * @return the avatar controller for displaying collaborators' avatars
+     */
+    public BoxAvatarView.AvatarController getAvatarController() {
```

Comment:
Who needs this?

Reply:
The CollaboratorInitialsView make use of this when it passed the controller to BoxAvatarView.